runtime/CMakeLists.txt

```diff
@@ -135,10 +135,19 @@ add_custom_target(j9vm_hookgen
 
 # Generate NLS headers
 file(GLOB_RECURSE nls_files "${CMAKE_CURRENT_SOURCE_DIR}/nls/*.nls")
+
+# On z/OS, nls generated files must be tagged as ASCII to allow for accurate EBCDIC conversion.
+if(OMR_OS_ZOS)
+	set(tag_nls_headers find ${CMAKE_CURRENT_BINARY_DIR}/nls -name "*.h" | xargs chtag -t -c ISO8859-1)
```

Comment:
I believe the canonical name is `ISO-8859-1`: That should be used here.

Reply:
Changing the name to `ISO-8859-1`:
```
FSUMF174 chtag: You must enter a codeset name or a ccsid between 0 and 65535 (USHRT_MAX).
```
Are we sure this the proper way to tag them?